### DISCUSSION TITLE
is using RTE without @mantine/core and @mantine/hooks possible?

### DISCUSSION THREAD

**Discussion by Lunatix01** [mantinedev/mantine#2068] [Q&A]
### Hello there
I know these 2 packages are dependent for RTE, in my case I don't use Mantine just wanted the RTE I was wondering if I can install the RTE only. because I don't need to install extra stuff which I'm not using. thanks.

**Comment by rtivital** [comment]
Rich text editor depends on core and hooks packages, you cannot use it wihout them
